buildchain/buildchain/versions.py

```diff
@@ -228,132 +228,193 @@ def _version_prefix(version: str, prefix: str = 'v') -> str:
 CONTAINER_IMAGES_MAP = {image.name: image for image in CONTAINER_IMAGES}
 
 # }}}
+
 # Packages {{{
 
+class PackageVersion:
+    """A package's authoritative version data.
+
+    This class contains version information for a named package, and
+    provides helper methods for formatting version/release data as well
+    as fully versioned name, for all supported OS families.
+    """
+
+    def __init__(
+        self,
+        name: str,
+        version: Optional[str] = None,
+        release: Optional[str] = None
+    ):
+        """Initializes a package version.
+
+        Arguments:
+            name: the name of the package
+            version: the version of the package
+            release: the release of the package
+        """
+        self._name = name
+        self._version = version
+        self._release = release
+
+    name = property(operator.attrgetter('_name'))
```

Comment:
minor: would align

```
    name    = property(operator.attrgetter('_name'))
    version = property(operator.attrgetter('_version'))
    release = property(operator.attrgetter('_release'))
```

Reply:
Damn that triggers my linter like crazy. Done.